charts/tempo/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: tempo
 description: Grafana Tempo Single Binary Mode
 type: application
-version: 1.11.0
-appVersion: 2.5.0
+version: 1.11.1
```

```suggestion
version: 1.12.1
```
@zanhsieh how about we bump it to `1.12.1` since the  `app version: appVersion: 2.6.1` introduces minor as well as patch version change ?

---

Sure.